src/dsp_tools/commands/ingest_xmlupload/bulk_ingest_client.py

```diff
@@ -82,3 +84,36 @@ def upload_file(
         print(f"{datetime.now()}: {msg}")
         logger.info(msg)
         return None
+
+    def kick_off_ingest(self) -> None:
+        """Start the ingest process on the server."""
+        url = f"{self.dsp_ingest_url}/projects/{self.shortcode}/bulk-ingest"
+        res = self.session.post(url, timeout=5)
+        if res.status_code == STATUS_CONFLICT:
+            msg = f"Ingest process on the server {self.dsp_ingest_url} is already running. Wait until it completes..."
+            print(msg)
+            logger.info(msg)
+        if res.json().get("id") != self.shortcode:
+            raise UserError("Failed to kick off the ingest process.")
```

Comment:
Why not InputError? From what I remember we use InputError when we create a new error in the code.

Reply:
If I compare the docstring of `UserError`, it seems more appropriate than `InputError`. And also the names of these two suggest that this here is rather a UserError. 

I don't think that every new error that we raise in the code should be an InputError. I rather think that we should tidy up our exception hierarchy in `src/dsp_tools/models/exceptions.py`, that's a big mess, and it lacks a clear concept. I have added it to our Topics To Discuss list